### DISCUSSION TITLE
The sitemap.xml

### DISCUSSION THREAD

**Discussion by xxl1212** [fastapi/fastapi#6691] [Questions] [answered]
Hi,

Is it possible to automatically generate the sitemap.xml? If yes, how ?

PS: Sorry, write with Google Translate (https://translate.google.com)

**Comment by ycd** [comment, marked as answer]
No, FastAPI doesn't have this feature by default, however you can generate sitemap by iterating over attached paths to your FastAPI instance.

**Comment by xxl1212** [comment]
Thank you !

**Comment by tiangolo** [comment]
Thanks for the help @ycd! 🤓 And thanks for coming back to close it @xxl1212 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
